shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/plugin/interceptor/ClassStaticMethodAroundInterceptor.java

```diff
@@ -40,6 +41,8 @@
     
     private final ClassStaticMethodAroundAdvice classStaticMethodAroundAdvice;
     
+    private boolean tryCall = true;
```

Comment:
is it better to rename `tryCall` as `needCheck`?

Reply:
TryCall is the flag to call the advice.  Maybe it can rename to needCall